go-controller/.golangci.yml

```diff
@@ -28,6 +29,23 @@ linters-settings:
     disable:
       - fieldalignment
       - shadow
+  
+  importas:
+    no-unaliased: true
+    alias:
+      # Kubernetes
+      - pkg: k8s.io/api/core/v1
+        alias: corev1
+      - pkg: k8s.io/apimachinery/pkg/apis/meta/v1
+        alias: metav1
+      - pkg: k8s.io/apimachinery/pkg/api/errors
+        alias: apierrors
+      - pkg: k8s.io/apimachinery/pkg/util/errors
+        alias: kerrors
+      # Controller Runtime
+      - pkg: sigs.k8s.io/controller-runtime
+        alias: ctrl
```

Comment:
There's a lot of inconsistency with `"github.com/ovn-org/libovsdb/ovsdb"` too (`libovsdb` vs `ovsdb`)

Reply:
Will add rule